app/misc.py

```diff
@@ -229,6 +233,14 @@ def get_global_stylesheet(self):
         return ''
 
 
+def is_user_admin(uid):
+    get_admin_users = User.select(User.uid).join(UserMetadata).where((UserMetadata.key == 'admin') & (UserMetadata.value == '1'))
+    admin_user_list = [user.uid for user in get_admin_users]
+
+    for admin_user in admin_user_list:
+        if uid in admin_user:
+            return True
+
```

Comment:
This could be optimized by filtering with `UserMetadata.uid == uid & UserMetadata.key == 'admin' ....`, then you can use `.get()` to fetch the record (and catch the `UserMetadata.DoesNotExist` exception when the user is not an admin).

That way you don't fetch unnecessary data and avoid the for altogether

Reply:
Sounds good.